README.md

```diff
@@ -75,4 +75,4 @@ docker-compose up -d
 - Run example in sbt, for example: 
 ```
 examples/runMain example3.CatsEffectApp
-```
\ No newline at end of file
+```
```

Comment:
I assume this is a newline thing?

Reply:
Yep.  Ancient POSIX rule.  It's a good change.